Containers/Home/Home.js

```diff
@@ -177,10 +177,9 @@ const styles = StyleSheet.create({
     width: '100%',
   },
   map: {
-    left: '5%',
-    width: '90%',
-    height: '70%',
-    top: -100,
+    width: '100%',
+    height: '77%',
+    top: -110,
```

Comment:
Can you try making this top: '-23%', currently it doesn't look good on the iPhone xR.

Reply:
Had a small white line so I changed it to -24%

Reply:
Ok, thanks. Looks great!